lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -1018,17 +1018,23 @@ SwiftLanguage::GetHardcodedSynthetics() {
           struct IsEligible {
```

Comment:
I know this isn't your code, but it's hard to understand what "IsEligible" is checking for. Could you comment this function, either on top or inside it explaining what the eligibility criteria are?

Also, this structure with static function could be replaced by a lambda.